manifests/on-prem/coredns-corefile.tmpl

```diff
@@ -8,7 +8,7 @@
     reload
     template IN {{`{{ .Cluster.IngressVIPRecordType }}`}} {{ .ControllerConfig.DNS.Spec.BaseDomain }} {
         match .*.apps.{{ .ControllerConfig.DNS.Spec.BaseDomain }}
-        answer "{{`{{"{{ .Name }}"}}`}} 60 in {{`{{"{{ .Type }}"}}`}} {{ onPremPlatformIngressIP .ControllerConfig }}"
+        answer "{{`{{"{{ .Name }}"}}`}} 60 in {{`{{"{{ .Type }}"}}`}} {{ index (onPremPlatformIngressIPs .ControllerConfig) 0 }}"
```

Comment:
Yeah, I was lazy with this one since during bootstrap we just care about being able to resolve the endpoints, we don't need dual stack capability at that point.